python/src/opendp/context.py

```diff
@@ -283,14 +286,15 @@ def _range_warning(name, value, info_level, warn_level):
 
 def unit_of(
     *,
-    contributions: Optional[int] = None,
+    contributions: Optional[Union[int, Sequence[Bound]]] = None,
     changes: Optional[int] = None,
     absolute: Optional[float] = None,
     l1: Optional[float] = None,
     l2: Optional[float] = None,
+    identifier: Optional[str] = None,
     ordered: bool = False,
     U=None,
-) -> tuple[Metric, float]:
+) -> tuple[Metric, Union[float, Sequence[Bound]]]:
     """Constructs a unit of privacy, consisting of a metric and a dataset distance. 
     The parameters are mutually exclusive.
```

Comment:
If it's ok to think of `Sequence[Bound]` as a dataset distance, then the status quo is fine?

Reply:
Yep. It's technically a multiple bounds on dataset distances, but if you squint hard enough it looks the same.